content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -53,24 +53,25 @@ creating a patch to fix it in the upstream project.
 
 ## Cloning the Kubernetes repository
 
-If you don't already have the kubernetes/kubernetes repository, get it now:
```

Comment:
I don't think this needs changing, tbh.

Reply:
Hi @divya-mohan0209,thanks for the review, originally both comment seems similar that's why I removed above line.

Reply:
I will add the above line in the updated PR.